src/arch/arm/32/kernel/vspace.c

```diff
@@ -1213,7 +1213,7 @@ exception_t handleVMFault(tcb_t *thread, vm_fault_type_t vm_faultType)
 
 #ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
         addr = getHDFAR();
-        addr = (addressTranslateS1CPR(addr) & ~MASK(PAGE_BITS)) | (addr & MASK(PAGE_BITS));
+        addr = (addressTranslateS1(addr) & ~MASK(PAGE_BITS)) | (addr & MASK(PAGE_BITS));
```

Comment:
@yyshen, do you think we should add `if (ARCH_NODE_STATE(armHSVCPUActive)) {` here to avoid a no-op with `addressTranslateS1`?